### PR TITLE
adding option --edge to install latest edge / preview version

### PR DESCRIPTION
**Pull request by Leask**
Adding option --edge to install latest edge / preview version.

```bash
$ ./install.sh --edge true
```

```bash
$ ./install.sh --edge=true
```

### CODE REVIEW

File: install.sh [hunk 1]
```diff
@@ -32,6 +32,9 @@ Usage:
   --version X.X.X
       Install a specific version instead of the latest.
 
+  --edge [true | false (default) ]
```
Comment (code-asher) [inline install.sh:35]: Could we leave off the value and set this to true when the flag is set?  This is how we have been handling booleans, for example see `--dry-run`.

**Review by code-asher — DISMISSED** [review]
This is a fantastic idea!

**Comment by Leask** [comment]
Adding this commit: `updated arg --edge parsing to match the style of --dryrun`
https://github.com/cdr/code-server/pull/4605/commits/746f55a2b8046aea5eef0d2e13bec3a134b7d603
Thanks for the review :)

**Review by jsjoeio — APPROVED** [review]
Woohoo! Thanks so much

**Comment by jsjoeio** [comment]
(You can ignore Docs preview - I need to fix it)

**Comment by Leask** [comment]
Thanks!

**Comment by Leask** [comment]
🆒

**Pull request merged**